src/BinSkim.Driver/AnalyzeCommand.cs

```diff
@@ -85,6 +85,17 @@ public override int Run(AnalyzeOptions analyzeOptions)
                 analyzeOptions.Kind = new List<ResultKind> { ResultKind.Fail, ResultKind.NotApplicable, ResultKind.Pass };
             }
 
+#pragma warning disable CS0618 // Type or member is obsolete
+            if (analyzeOptions.ComputeFileHashes)
```

Comment:
>analyzeOptions.ComputeFileHashes [](http://example.com/codeflow?start=16&length=32)

this change is needed to prevent a breaking change in the guardian tasks.
We need to support --Hashes

Reply:
maybe we should make this change in SDK instead?
AnalyzeCommandBase has this check,
MultithreadedAnalyzeCommandBase does not,

if the change is made in Binskim then only Binskim is fixed but not SDK

Reply:
I was wondering about that.
Since we want to deprecate --hashes, i would move that logic to AnalyzeCommand and remove the reference from AnalyzeCommandBase.

With that, in the futhre, the options would not have --hashes by default and binskim itself would implement it.